com.unity.multiplayer.mlapi/Runtime/Profiling/IHasProfilableTransport.cs

```diff
@@ -0,0 +1,7 @@
+namespace MLAPI.Profiling
+{
+    public interface IHasProfilableTransport
```

Comment:
what about `IProfilableTransport`, `ITransportProfileProvider` or `ITransportProfileSource` etc.? @kvassall-unity @becksebenius-unity — I'm just trying to nit-pick on the "Has" part :D

Reply:
I don't mind the `Has` personally, but it does sound gramatically awkward - what about `IProfilableTransportProvider` (kind of a combination of your suggestions)

Reply:
sounds good to me! :)

Reply:
@kvassall-unity 👀